application/views/main/messages/compose.php

```diff
@@ -4,23 +4,36 @@
 <script language="javascript" src="<?php echo $this->config->item('js_path');?>jquery-plugin/jquery.tokeninput.min.js"></script>
 <script language="javascript" src="<?php echo $this->config->item('js_path');?>jquery-plugin/jquery.form.min.js"></script>
 <style type="text/css">
-.left_aligned { margin-left:0; padding-left:0;}
-.form_option { display:flex }
-.opt { flex-grow: 1}
+	.left_aligned
+	{
+		margin-left: 0;
+		padding-left: 0;
+	}
+
+	.form_option
+	{
+		display: flex
+	}
+
+	.opt
+	{
+		flex-grow: 1
+	}
+
```

Comment:
can we not beautify css which are inline/within the document. This just adds too much to white space to the document.

Reply:
you mean disable beautification in that case?

Reply:
I found [some options in html-beautify](https://github.com/beautify-web/js-beautify/#css--html):
- List of tags (defaults to pre) whose content should not be reformatted. We can put `style` in this list. This will never be touched. (this is the `-T` option)
- `--newline-between-rules false` as it says
- `--brace-style=collapse`  (which will do the same in JS also) will give this
```
rrr {
}
```

The way CSS is formatted may be a bit surprising but at least it permits to immediately know where the CSS is.

[There are also](https://github.com/beautify-web/js-beautify/blob/main/js/src/css/options.js):
selector_separator_newline
space_around_selector_separator
space_around_combinator

Any preference?

Reply:
`--brace-style=collapse` makes more sense. IMO we should enable this. Same with `--newline-between-rules false` this will keep the code still usable and remove unnecessary lines.

Also `selector-combinator-space-after` & `space_around_combinator` looks sensible to me as that just adds single space.

Reply:
What do you mean by sensible?

Reply:
I meant we should use that. Can you enable `--brace-style=collapse` , `--newline-between-rules false`, `selector-combinator-space-after` , `space_around_combinator`

Reply:
Done. selector-combinator-space-after didn't exist. I used `--space-around-selector-separator` instead